packages/links-mysql/links-mysql.0.9.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "James Cheney <jcheney@inf.ed.ac.uk>"
+authors: "The Links Team <links-dev@inf.ed.ac.uk>"
+synopsis: "MySQL database driver for the Links Programming Language"
+description: "MySQL database driver for the Links Programming Language"
+homepage: "https://github.com/links-lang/links"
+dev-repo: "git+https://github.com/links-lang/links.git"
+bug-reports: "https://github.com/links-lang/links/issues"
+license: "GPL-2"
+
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "1.10.0"}
+  "conf-mysql"
```

Comment:
the `mysql` package already has the dependency on `("conf-mariadb" | "conf-mysql")`. I would encourage you to either remove this dependency or copy it over (both are equivalent), unless you specifically require `mysql` and not `mariadb`, for next time

Reply:
Good point! However, this is intentional, as we don't test against mariadb in our current CI setup and would rather be conservative about the databases we claim to support.